src/pages/home/_hero.js

```diff
@@ -182,7 +182,7 @@ export const Hero = () => {
                             </CheckMarkBullet>
                         </CheckMarkList>
                         <ButtonWrapper jc="unset">
-                            <HeroButton type="submit" secondary="true" to="/signup/">
+                            <HeroButton type="submit" secondary to="/signup/">
```

Comment:
In the latest code reviews I was told to add the value for syled-component: secondary="true".
Which one is correct? @negarn @oskar-binary

Reply:
I see, in deriv-app we have the styleguide to not add true values to boolean parameters as they will be true by default, so I was following that. please keep the true then.

Reply:
@negar-binary for react components yes but not for styled-components, will check if there is any way to get around the warning message.